### ISSUE TITLE
Question: Missing documentation for Active Organization?

### ISSUE THREAD

**Issue by michaeljauk** [p2-inc/phasetwo-docs#173]
Hey there,

I hope this is the right place to ask this (otherwise feel free to delete this/point me in the right direction).

Is there any reason why there is no documentation (and no updated API docs) available for the Active Organizations feature as described in https://github.com/p2-inc/keycloak-orgs/blob/main/docs/active-organization.md, since it seems this feature is generally available?

This is actually I guess very useful to a lot of users, so it'd be kinda sad that many people are probably not aware of it because of missing documentation

**Comment by xgp** [comment]
Good catch. This hasn't been added to the main docs, as we're not officially "supporting" this in our commercial hosting system yet. We will add it once we have.
